src/pages/using-spark/guides/validation-messaging.mdx

```diff
@@ -0,0 +1,106 @@
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Main Takeaways
+</SprkHeading>
+
+Giving validation feedback is an emotional process. Support users with this process by following these guidelines:
+
+- Explain what the Input expects before the user enters data. (Example: password rules, ranges, etc)
+- Give clear and helpful feedback when encountering a problem.
+- Use a combination of [placement](#placement), [timing](#timing), and [language](#language) to guide users through moments of uncertainty.
+
+<img
+  srcSet={validationAnatomySet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbh"
+  src={validationAnatomy}
+  alt="Anatomy of input validation with numbers pointing to each part: timing, timing, and language."
+/>
+
+
+<span id="placement" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Placement
+</SprkHeading>
+
+- Place messaging close to the input. Spark places validation directly under the input.
+- Provide a clear visual to indicate something has gone wrong with user input and needs attention.
+- Plan error messaging for every field.
+- On Forms with multiple fields, show validation messaging for each input field to guide users to a successful submission.
+
+<span id="timing" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Timing
+</SprkHeading>
+
+-	Show validation messaging after the user enters the data or when the input becomes out of **selection focus**.
+- On forms, show error messaging **before** the user submits the form.
+
+> **Selection focus** changes when a user taps or clicks anywhere outside of the current focused input. Focus can also change
+> through keyboard navigation.
+
+<span id="language" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Language
+</SprkHeading>
+
+<ul>
+  <li>
+    Use labels and helper text to clarify what data the input expects. Messaging should be related and relevant to the Input.
+  </li>
+  <li>
+    Deliver direct and concise statement of the problem in non-technical terms. Write out abbreviations.
+  </li>
+  <li>
+    Use sentence case. Do not use all caps.
+  </li>
+  <li>
+    Direct users to the next action to solve the problem.
+    <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+      <li>
+        Use positive action verbs, rather than negative ones. Focus on the solution by explaining what to do, opposed to what not to do.
+      </li>
+      <li>
+        If the “error” is a system condition that blocks the user from accomplishing a task, offer them a way to be notified of updated system status.
+      </li>
+    </ul>
+  </li>
+  <li>
+    Follow accessibility guidelines, and where applicable, ARIA-live attributes.
+  </li>
+</ul>
```

Comment:
Nested lists look really weird rendered through markdown, we went through this in color pages, so I copied the solution there and added necessary classes